pyproject.toml

```
@@ -2,6 +2,74 @@
 build-backend = "hatchling.build"
 requires = ["hatch-vcs", "hatchling"]
 
+[dependency-groups]
+dev = ["mne[doc,test]", "rcssmin"]
```

this is not the correct syntax for group inheritance. 

```suggestion
dev = ["rcssmin", { include-group = "doc" }, { include-group = "test" }]
```

https://packaging.python.org/en/latest/specifications/dependency-groups/#dependency-group-include